silo/tests/test.py

```diff
@@ -209,6 +210,43 @@ def test_silo_detail_import_done(self):
         self.assertNotContains(response, "<h4>Import process running</h4>")
 
 
+class OneDriveReadTest(TestCase):
+    fixtures = ['../fixtures/read_types.json']
+    new_read_url = '/source/new/'
+
+    def setUp(self):
+        self.client = Client()
+        self.factory = RequestFactory()
+        self.tola_user = factories.TolaUser()
+
+    def test_new_read_post(self):
+        read_type = ReadType.objects.get(read_type="OneDrive")
+
+        params = {
+            'owner': self.tola_user.user.pk,
+            'type': read_type.pk,
+            'read_name': 'TEST READ ONEDRIVE',
+            'description': 'TEST DESCRIPTION for test read source',
+            'onedrive_file': 'TEST10000100',
+            'onedrive_access_token':'TEST_DUMMY_TOKEN',
+            'create_date': '2018-01-26 12:33:00',
+        }
+        request = self.factory.post(self.new_read_url, data = params)
+        request.user = self.tola_user.user
+
+        response = showRead(request, 1)
```

Comment:
To make sure that in the `showRead` endpoint no `Read` will be found, use 0 as the id.